Sources_v3/StreamChat/Workers/ChannelUpdater.swift

```diff
@@ -25,7 +25,12 @@ class ChannelUpdater<ExtraData: ExtraDataTypes>: Worker {
                 channelCreatedCallback?(payload.channel.cid)
                 self.database.write { (session) in
                     try session.saveChannel(payload: payload)
-                    completion?(nil)
+                } completion: { error in
+                    if let error = error {
+                        completion?(error)
```

Comment:
either you forgot `log` here, or this unwrap do nothing

Reply:
Fixed